CONTRIBUTING.md

```diff
@@ -37,7 +37,7 @@ If you are modifying any of the Python code in this project, please use [Black](
 black lib --line-length 79
 ```
 
-The [CI](https://github.com/apache/fluo-muchos/tree/main/.travis.yml) for this project runs tools to detect common coding issues with Python and Ansible files. Rather than wait for the CI to flag any issues with your work, please run the [cibuild](https://github.com/apache/fluo-muchos/tree/main/scripts/cibuild.sh) script on your dev machine, which in turn runs the following tools:
+The [CI](./.github/workflows/ci.yaml) for this project runs tools to detect common coding issues with Python and Ansible files. Rather than wait for the CI to flag any issues with your work, please run the [cibuild](https://github.com/apache/fluo-muchos/tree/main/scripts/cibuild.sh) script on your dev machine, which in turn runs the following tools:
```

Comment:
I'm not sure all the views where this file is shown, but it might be better to use an absolute path, rather than relative to this file, in case it is shown in views other than the code tree browser.